src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -661,9 +661,15 @@ public function isPublic()
      * @return Definition The current instance
      *
      * @api
+     *
+     * @deprecated since version 2.7, will be removed in 3.0.
      */
-    public function setSynchronized($boolean)
+    public function setSynchronized($boolean, $triggerDeprecationError = true)
```

Comment:
Isn't it a BC break to add an extra argument to a public method signature? The method is flagged with `@api`...